_data/alan-wake-ii.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
can we keep all json files related to scores under folder `scores`?

Reply:
No, the json file must be saved in _data to be readable